CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 ## [0.6.1]
+* Removes blank area above sidebar contents, this can be set by developer using padding or container in builder
+* Adds optional top attribute, like the bottom attribute, to Sidebar.
+* Makes text alignment avaialble for title and message in MacosAlertDialog
+* Makes icon optional for MacosAlertDialog
```

Comment:
Icon should be required, per Apple guidelines

Reply:
Yeah, I realize that, but even Apple isn't always strict to ones like this, can remove tho

Reply:
Please do - even if Apple isn't strict, this is what their documentation says. I've also not yet seen an Alert dialog on Big Sur that doesn't have an icon. So the icon must be required.

Reply:
Any movement on this?

Reply:
No, and my apologies. I do not have time, and I'm going to be moving away from macos_ui. It's a killer framework and thanks for all you have done.